plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { useAsync } from 'react-use';
+import {
+  useApi,
+  Progress,
+  Content,
+  Header,
+  Page,
+  ErrorPage,
+} from '@backstage/core';
+import { scaffolderApiRef } from '../../api';
+import {
+  Typography,
+  Paper,
+  Table,
+  TableBody,
+  Box,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  makeStyles,
+} from '@material-ui/core';
+import { JSONSchema } from '@backstage/catalog-model';
+import { JSONSchema7Definition } from 'json-schema';
+
+const useStyles = makeStyles(theme => ({
+  code: {
+    fontFamily: 'Menlo, monospaced',
+    padding: theme.spacing(1),
+    backgroundColor:
+      theme.palette.type === 'dark'
+        ? theme.palette.grey[700]
+        : theme.palette.grey[300],
+    display: 'inline-block',
+    borderRadius: 5,
+    border: `1px solid ${theme.palette.grey[500]}`,
+    position: 'relative',
+  },
+
+  codeRequired: {
+    '&::after': {
+      position: 'absolute',
+      content: '"*"',
+      top: 0,
+      right: theme.spacing(0.5),
+      fontWeight: 'bolder',
+      color: theme.palette.error.light,
+    },
+  },
+}));
+
+export const ActionsPage = () => {
+  const api = useApi(scaffolderApiRef);
+  const classes = useStyles();
+  const { loading, value, error } = useAsync(async () => {
+    return api.listActions();
+  });
+
+  if (loading) {
+    return <Progress />;
+  }
+
+  if (error) {
+    return (
+      <ErrorPage
+        statusMessage="Failed to load installed actions"
+        status="500"
+      />
```

Comment:
ou this is a new one to me.  different than using `<WarningPanel severity="error">`.  maybe another one to align to https://github.com/backstage/backstage/issues/3781 :)

Reply:
Good to see an RFC for this. I believe the ErrorPage make sense when the initial content can't be displayed.